Automattic/wp-calypso
# Command palette: Add more commands (1st batch)

Part of https://github.com/Automattic/dotcom-forge/issues/7510

## Proposed Changes

Adds a subset of the missing commands identified in https://github.com/Automattic/dotcom-forge/issues/7510#issuecomment-2139811632:

- Open hosting overview
- Manage emails
- Open Jetpack Search
- Open Jetpack Akismet Anti-spam
- Manage categories
- Manage tags
- Open site editor
- Manage add-ons
- Manage media settings
- Open marketing tools         

## Why are these changes being made?

Because we want all menus to be accessible from a command

## Testing Instructions

- Use the Calypso live link below
- Open the command palette
- Select one of the commands above
- Make sure it links to the expected page
- Test it for a variety of sites: Simple Default, Simple Classic, Atomic Default, Atomic Classic, Jetpack, P2, Staging
- Repeat with the rest of the commands
